hr_holidays_public/__manifest__.py

```diff
@@ -0,0 +1,27 @@
+# Copyright 2015 2011,2013 Michael Telahun Makonnen <mmakonnen@gmail.com>
+# Copyright 2019 Camptocamp
```

Comment:
```suggestion
```

Reply:
@emagdalenaC2i any reason? IMHO each work can be credited no?

Reply:
Yeah, of course. But as contributor, IMHO not to be included in the copyright for a simple migration